lib/matplotlib/text.py

```diff
@@ -301,16 +301,16 @@ def set_rotation_mode(self, m):
 
         Parameters
         ----------
-        m : {None, 'default', 'anchor'}
+        m : {None, 'default', 'anchor', 'xtick', 'ytick'}
             If ``"default"``, the text will be first rotated, then aligned according
-            to their horizontal and vertical alignments.  If ``"anchor"``, then
-            alignment occurs before rotation. Passing ``None`` will set the rotation
-            mode to ``"default"``.
+            to their horizontal and vertical alignments.  If ``"anchor"``, ``"xtick"``
+            or ``"ytick", then alignment occurs before rotation. Passing ``None`` will
+            set the rotation mode to ``"default"``.
```

Comment:
```suggestion
            to their horizontal and vertical alignments.  If ``"anchor"``, then
            alignment occurs before rotation. "xtick" and "ytick" adjust the
            horizontal/vertical alignment so that the text is visually pointing
            towards its anchor point. This is primarily used for rotated tick
            labels and positions them nicely towards their ticks. Passing
            ``None`` will set the rotation mode to ``"default"``.
```